kythe/cxx/extractor/cxx_extractor_bazel_main.cc

```diff
@@ -28,11 +28,15 @@
 #include "google/protobuf/io/zero_copy_stream.h"
 #include "google/protobuf/io/zero_copy_stream_impl.h"
 #include "google/protobuf/stubs/common.h"
+#include "kythe/cxx/common/path_utils.h"
 #include "kythe/cxx/extractor/language.h"
 #include "third_party/bazel/src/main/protobuf/extra_actions_base.pb.h"
 
 DEFINE_string(build_config, "",
               "Human readable description of the build configuration.");
+DEFINE_string(canonicalize_vname_paths, "clean-only",
+              "Policy to use when canonicalization VName paths: "
+              "clean-only (default), prefer-relative, prefer-real.");
```

Comment:
It's super nitpicky, but is there a reason you went with dashes instead of underscores (like most flags use)?

Reply:
Flag names typically use underscores (because they flag library requires it); flag values are substantially more varied in the separator.  Clang/GCC flags, for example, universally use dashes.  Similarly, underscores are typical when the flags in question are auto-generated from ALL_CAPS enumerators, but much less consistent when that's not the case.

The short is I went with dashes because I think they're more readable, but am not particularly wedded to it.

Reply:
SGTM. I also think dashes are easier to read.